debugging/coreclr/dockerClient.ts

```diff
@@ -193,6 +195,8 @@ export class CliDockerClient implements DockerClient {
             .withArrayArgs('--env-file', options.envFiles)
             .withKeyValueArgs('--label', options.labels)
             .withArrayArgs('-v', options.volumes, volume => `${volume.localPath}:${volume.containerPath}${volume.permissions ? ':' + volume.permissions : ''}`)
+            .withArrayArgs('-p', options.ports)
```

Comment:
While it may happen that adding an explicit mapping via `-p` will override an earlier implicit mapping via `-P`, it would probably be better to instead just switch between using `-p` and `-P` depending on whether ports have been explicitly mapped, or not.

Reply:
As i've said in a comment earlier, I'll check for the existence of port mapping configuration so the command only binds those.